src/common/Pagination/styled.js

```diff
@@ -1,46 +1,154 @@
-import styled, { css } from "styled-components";
+import styled, { css, keyframes } from "styled-components";
+import { ReactComponent as arrow } from "./images/previousArrow.svg";
+import { ReactComponent as arrowNext } from "./images/nextArrow.svg";
 
 export const PageChanger = styled.div`
+  margin: 40px 0px 87px 0px;
   display: flex;
-  margin: 40px 0 103px;
   justify-content: center;
+  align-items: center;
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.large}) {
+    margin: 32px 0px 87px 0px;
+  } 
+  @media (max-width: ${({ theme }) => theme.breakpoints.small}) {
+    margin: 32px 0px 15px 0px;
+  } 
+`;
+
+const move = keyframes`
+  0% {
+    transform: translateX(0px);
+  }
+  25% {
+    transform: translateX(-3px);
+  }
+  50% {
+    transform: translateX(0px);
+  }
+  75% {
+    transform: translateX(3px);
+  }
+  100% {
+    transform: translateX(0px);
+  }
 `;
```

Comment:
Szczerze mowiac, to przerzucilbym ta stala, blizej tego gdzie jest uzywana, czyli dalbym ja nad ArrowIcon